contracts/script/OPSuccinctDGFDeployer.s.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {Script} from "forge-std/Script.sol";
+import {OPSuccinctDisputeGame} from "../src/OPSuccinctDisputeGame.sol";
+import {OPSuccinctDisputeGameFactory} from "../src/OPSuccinctDisputeGameFactory.sol";
+import {Utils} from "../test/helpers/Utils.sol";
+import {Proxy} from "@optimism/src/universal/Proxy.sol";
+import {console} from "forge-std/console.sol";
+
+contract OPSuccinctDFGDeployer is Script, Utils {
+    function run() public returns (address) {
+        vm.startBroadcast();
+
+        address l2OutputOracleProxy = vm.envAddress("L2OO_ADDRESS");
+
+        // This initializes the dipute game
```

Comment:
Fix: Initialize the dispute game based on the existing L2OO_ADDRESS. Should we always deploy the L2OO before creating the fault game implementation. Should the L2OO_ADDRESS be the proxy address.

Reply:
Not sure to understand, are you asking to update the comment?

Reply:
Haha sorry, can you update the comment to:

Initialize the dispute game based on the existing L2OO_ADDRESS.